src/openrct2/localisation/LanguagePack.cpp

```diff
@@ -651,7 +659,29 @@ class LanguagePack final : public ILanguagePack
     std::string FixRTL(std::string& input)
     {
 #ifdef _WIN32
-        return input;
+        FriBidiChar utf32String[1024] = { 0 };
```

Comment:
Can we use a (local) constant for this `1024`? It's used several times throughout this function.